patches/server/0005-Paper-config-files.patch

```diff
@@ -5111,6 +5111,47 @@ index a6d2ce801b236b046b94913bccf7eccfc561f35a..b244b65799d9c082b8b1639bad157274
      }
  
      public <T> List getList(String path, T def)
+@@ -138,14 +144,14 @@ public class SpigotWorldConfig
+     public double itemMerge;
+     private void itemMerge()
+     {
+-        this.itemMerge = this.getDouble("merge-radius.item", 2.5 );
++        this.itemMerge = this.getDouble("merge-radius.item", 0.5 );
+         this.log( "Item Merge Radius: " + this.itemMerge );
+     }
+ 
+     public double expMerge;
+     private void expMerge()
+     {
+-        this.expMerge = this.getDouble("merge-radius.exp", 3.0 );
++        this.expMerge = this.getDouble("merge-radius.exp", 0.5 );
```

Comment:
It was pointed out that the vanilla value for this is actually to disable it completely with `-1`. There are 2 exp merge systems in the server, this only controls the custom one spigot added, and doesn't mess with vanilla's own merge system. So disabling this completely will result in vanilla xp behavior.

Reply:
Oh I was not aware of this and yeah that would certainly be a more favorable result.